test/e2e/test_catalogue_item.py

```diff
@@ -1048,10 +1048,46 @@ def test_partial_update_catalogue_item_change_catalogue_category_id_without_prop
     assert response.status_code == 422
     assert (
         response.json()["detail"]
-        == "Invalid value type for catalogue item property 'Property A'. Expected type: boolean."
+        == "Cannot move catalogue item to a category with different catalogue_item_properties without specifying the "
+        "new properties"
     )
 
 
+def test_partial_update_catalogue_item_change_catalogue_category_id_with_properties(test_client):
```

Comment:
Isn't this a duplicate of `test_partial_update_catalogue_item_change_catalogue_category_id`?